content/guides/references/configuration.md

```diff
@@ -330,35 +330,10 @@ You can
 
 </Alert>
 
-### `Cypress.config()`
+## Test-time overrides
```

Comment:
I'm not sure we want to update the wording of this to 'Test-time overrides'. This change would require consulting with product/DX as this introduces brand new terminology to the product.

Reply:
I'll change